### DISCUSSION TITLE
All I need is deleting specific website's cookies and keep the rest.

### DISCUSSION THREAD

**Discussion by Kenan7** [Cookie-AutoDelete/Cookie-AutoDelete#1075] [Q&A / Support] [answered]
I whitelisted everything and after that greylisted the specific website I wanted.

is everything ok with these settings? I don't want to lose any website's cookies.

![image](https://user-images.githubusercontent.com/11595462/110461038-662e2400-80e8-11eb-9796-9f9cf1ce2a84.png)

Thank you.

**Comment by kennethtran93** [comment, marked as answer]
At this time we do not have the ability to remove specific website's data yet.  

Whitelist takes precedence over greylist, so if it matches on the whitelist side, then it gets kept.
